contrib/dev/Vagrantfile

```diff
@@ -2,23 +2,22 @@
 # vi: set ft=ruby :
 
 require 'vagrant-openstack-provider'
+require 'vagrant-reload'
+require 'vagrant-vbguest'
 
-PREPARE_BOX=ENV.fetch("PREPARE_BOX", "") == "true" ? true : false
-PROVISION_BOX=ENV.fetch("PROVISION_BOX", ENV.fetch("PREPARE_BOX", "")) == "true" ? true : false
-SYNCED_FOLDER=ENV.has_key?("GOPATH") ? ENV.fetch("GOPATH") + "/src/github.com/skydive-project/skydive" : "../.."
-
-$skydive_extra_config = {
-  "http.ws.pong_timeout" => 10,
-  "agent.topology.netlink.metrics_update" => 5,
-  "analyzer.listen" => "0.0.0.0:8082",
-  "analyzer.flow.backend" => "elasticsearch",
-  "analyzer.topology.backend" => "elasticsearch",
-  "storage.elasticsearch.host" => "127.0.0.1:9200"
-}
+PREPARE_BOX = ENV.fetch("PREPARE_BOX", "") == "true" ? true : false
+PROVISION_BOX = ENV.fetch("PROVISION_BOX", ENV.fetch("PREPARE_BOX", "")) == "true" ? true : false
+SKYDIVE_GITHUB = "/github.com/skydive-project/skydive"
+SYNCED_FOLDER = ENV.has_key?("GOPATH") ? ENV.fetch("GOPATH") + "/src" + SKYDIVE_GITHUB : "../.."
+GOPATH = "/home/vagrant"
```

Comment:
by default golang define gopath to $HOME/go

Reply:
overall what wasn't working to introduce so many changes ?
What's the goal of this patch ?

Reply:
the ansible code assumes GOPATH = "/home/vagrant" - this was implicit in the previous code - I simply made this explicit.